_includes/footer.html

```diff
@@ -4,12 +4,11 @@
           <a href="/terms-of-service/">Terms of Service</a>
         </nav>
         <p>
-          If you have questions or issues, it is always best to consult a legal professional.<br>
           The content of this site is licensed under the <a href="https://creativecommons.org/licenses/by/3.0/">
           Creative Commons Attribution 3.0 Unported License</a>.
         </p>
         <div class="with-love">
-          Demystified with &lt;3 by <a href="https://github.com">GitHub, Inc.</a>
+          Made with &lt;3 by <a href="https://github.com">GitHub, Inc.</a>
```

Comment:
Is there a better, more descriptive word than "demystified"? Published? Curated?